pkg/util/k8s/kubeproxy.go

```diff
@@ -81,6 +82,11 @@ func GenerateKubeProxyConfiguration(args map[string]operv1.ProxyArgumentList) (s
 
 	kpc.NodePortAddresses = ka.getCIDRList("node-port-addresses")
 
+	if kpc.FeatureGates == nil {
+		kpc.FeatureGates = make(map[string]bool)
+	}
+	kpc.FeatureGates[string(features.EndpointSlice)] = false
+	kpc.FeatureGates[string(features.EndpointSliceProxying)] = false
```

Comment:
We don't want to do this here; in particular, we eventually want to support running standalone kube-proxy with `IPv6DualStack=true` (and thus `EndpointSlice=true`). (SDN-1325)

You want to add these to `kpcOverrides` in `pkg/network/openshift_sdn.go`, though that will also require changing `MergeKubeProxyArguments`, because we want to merge the individual `FeatureGates` keys from each config source (defaults, user, overrides). And then we need to set `kpc.FeatureGates` correctly from `ka` here. This will need unit tests...

Reply:
Ack. I was a bit torn between doing it as per your proposal or the way I ended up implementing it generically for all kube-proxy configs. I was not aware of SDN-1235. I will change this per your proposal.